homeassistant/auth_providers/modules/insecure_example.py

```diff
@@ -0,0 +1,66 @@
+"""Example auth module."""
+import logging
+from collections import OrderedDict
+
+import voluptuous as vol
+
+from homeassistant import auth
+
+CONFIG_SCHEMA = auth.AUTH_PROVIDER_SCHEMA.extend({
+    vol.Required('users'): [vol.Schema({
+        vol.Required('username'): str,
+        vol.Required('pin'): str,
+    })]
+}, extra=vol.PREVENT_EXTRA)
+
+STORAGE_VERSION = 1
+STORAGE_KEY = 'auth_module.insecure_example'
```

Comment:
I think that we should store these on the user objects in a data object that is stored/restored when user is saved/restored.